create_dataset/combine_peaks_files.py

```diff
@@ -14,34 +16,62 @@
 def parse_args():
     parser = argparse.ArgumentParser(description="Combine individual HDF5 files into a single file.")
     parser.add_argument('recording_id', type=str, help="ID of the recording.")
-    parser.add_argument('num_tasks', type=int, default=1, help="Number of tasks used in processing.")
     return parser.parse_args()
 
 
 def main(args):
     data_folder = f'data/{args.recording_id}'
+    peaks_folder = os.path.join(data_folder, "peaks")
     
-    # Load peak data and channel locations
-    peaks = np.load(os.path.join(data_folder, "peaks/peaks_test.npy"))
+    # Load the original peak data and channel locations
+    peaks = np.load(os.path.join(peaks_folder, "peaks.npy"))
     channel_locations = np.load(os.path.join(data_folder, "channel_locations.npy"))
     
+    # Find all peaks files automatically
+    peaks_files = find_peaks_files(peaks_folder)
+    
     # Combine all the peak files
-    combine_peaks(peaks, channel_locations, args.num_tasks, data_folder)
+    combine_peaks_files(peaks, channel_locations, peaks_files, data_folder)
 
 
-def combine_peaks(peaks, channel_locations, num_tasks, data_folder):
+def find_peaks_files(folder):
+    """
+    Automatically find all peaks_X.h5 files in the given folder.
+    
+    Args:
+        folder: Path to the folder containing peak files.
+        
+    Returns:
+        List of paths to peak files.
+    """
+    peaks_files = []
+    pattern = re.compile(r'peaks_(\d+)\.h5$')
+    
+    for filename in os.listdir(folder):
+        if pattern.match(filename):
+            peaks_files.append(os.path.join(folder, filename))
```

Comment:
You can simplify this by using the standard library glob. Add `import glob` to the top and replace this with:

```suggestion
    # Find all peaks_*.h5 files
    peaks_files = glob.glob(os.path.join(folder, "peaks_*.h5"))
```

This will also get files that don't end in numbers, e.g., `peaks_ABC.h5`, but I think that's OK.